refinery/tool_manager/models.py

```diff
@@ -71,6 +78,16 @@ def get_galaxy_workflow_step(self):
                 GalaxyParameter.MultipleObjectsReturned):
             return None
 
+    def cast_param_value_to_proper_type(self, parameter_value):
```

Comment:
this seems odd but is necessary, docstring/ comments would be helpful. Also could utilize `Parameter.VALUE_TYPES` here